docs/source/parallel.rst

```diff
@@ -155,30 +161,67 @@ Parallel computation using the default joblib backend
     >>> print(parallel(delayed(expensive_calculation)(*a) for a in args))
     [1, 3, 5, 7, 9, 11, 13]
 
-You can do the the same parallel computation using the Civis backend
+You can do the same parallel computation using the Civis backend
 by creating and registering a backend factory and entering a
-``with parallel_backend('civis')`` context. The code below will start
+``with parallel_config('civis')`` context. The code below will start
 seven different jobs in Civis Platform (with up to five running at once).
 Each job will call the function ``expensive_calculation`` with a
-different set of arguments from the list ``args``.::
+different set of arguments from the list ``args``::
 
     >>> def expensive_calculation(num1, num2):
     ...     return 2 * num1 + num2
     >>> from joblib import delayed, Parallel
-    >>> from joblib import parallel_backend, register_parallel_backend
+    >>> from joblib import parallel_config, register_parallel_backend
     >>> from civis.parallel import make_backend_factory
     >>> register_parallel_backend('civis', make_backend_factory(
     ...     required_resources={"cpu": 512, "memory": 256}))
     >>> args = [(0, 1), (1, 1), (2, 1), (3, 1), (4, 1), (5, 1), (6, 1)]
-    >>> with parallel_backend('civis'):
-    ...    parallel = Parallel(n_jobs=5, pre_dispatch='n_jobs')
-    ...    print(parallel(delayed(expensive_calculation)(*a) for a in args))
+    >>> with parallel_config('civis'):
+    ...     parallel = Parallel(n_jobs=5, pre_dispatch='n_jobs')
+    ...     print(parallel(delayed(expensive_calculation)(*a) for a in args))
     [1, 3, 5, 7, 9, 11, 13]
 
-You can use the Civis joblib backend to parallelize any code which
+Since joblib v1.4.2, :class:`joblib.Parallel` has a ``return_as`` parameter
```

Comment:
Preview now available from [the doc build](https://civis-python--488.org.readthedocs.build/en/488/parallel.html)